tasks/mina/rails.rb

```diff
@@ -5,11 +5,11 @@
 set :bundle_prefix, -> { %{RAILS_ENV="#{fetch(:rails_env)}" #{fetch(:bundle_bin)} exec} }
 set :rake, -> { "#{fetch(:bundle_prefix)} rake" }
 set :rails, -> { "#{fetch(:bundle_prefix)} rails" }
-set :compiled_asset_path, 'public/assets'
-set :asset_dirs, ['vendor/assets/', 'app/assets/']
+set :compiled_asset_path, ['public/assets', 'public/packs']
+set :asset_dirs, ['vendor/assets/', 'app/assets/', 'app/javascript/']
```

Comment:
```suggestion
set :asset_dirs, ['vendor/assets/', 'app/assets/', 'app/javascript/', 'package.json']
```

Assets should recompile too when dependencies in package.json change.

Reply:
@lovro-bikic Applied suggested changes.